api/v1beta2/flowcollector_types.go

```diff
@@ -521,16 +521,15 @@ type LokiManualParams struct {
 	// https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network.
 	IngesterURL string `json:"ingesterUrl,omitempty"`
 
-	//+kubebuilder:validation:optional
-	// `querierURL` specifies the address of the Loki querier service, in case it is different from the
-	// Loki ingester URL. If empty, the URL value is used (assuming that the Loki ingester
-	// and querier are in the same server). When using the Loki Operator, do not set it, since
-	// ingestion and queries use the Loki gateway.
+	//+kubebuilder:default:="http://loki:3100/"
+	// `querierUrl` specifies the address of the Loki querier service.
+	// When using the Loki Operator, set it to the Loki gateway service with the `network` tenant set in path, for example
+	// https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network.
```

Comment:
Should we still mention loki operator in `manual` since we introduced `LokiStack` mode ?

Reply:
At first I removed them, then I thought perhaps there will be cases where people use loki operator but for some reason need to use manual params (e.g. using an old version or on the contrary a newer version which triggers changes in our lokistack mode handling) .. so perhaps it's doesn't hurt to keep this?

Reply:
sure :+1: